content/yubikey-login.md

```diff
@@ -15,98 +15,96 @@ tableOfContents: true
 ---
 
 
-If you have a Yubikey, you can use it to login or unlock  your system.
+If you have a Yubikey, you can use it to login or unlock your system.
 
-To do this you must install the yubikey-luks package, configure a challenge-response slot on the Yubikey, and then configure the necessary PAM modules.
+To do this you must install the yubikey packages, configure a challenge-response slot on the Yubikey, and then configure the necessary PAM modules.
 
-**NOTE:** This guide assumes you are running Pop!_OS
+**NOTE:** Open an additional root terminal: `sudo su`
 
 ## Install Packages
 
 To install the necessary packages, run:
 
-    sudo apt install -y libpam-yubico yubikey-personalization
-
-You may get a question about the PAM configuration line. If so, enter this line:
-
-    mode=challenge-response
-
-Don't enable any PAM modules yet.  We'll do that at the end.
-
-If you have already installed the package or want to reconfigure it, use this command:
-
-    sudo dpkg-reconfigure libpam-yubico
+    sudo apt install -y libpam-yubico yubikey-personalization yubikey-manager
 
 ## Configure Challenge-Response for your Yubikey
 
-To enable challenge-response on your Yubikey, type the following command:
+To enable challenge-response on your Yubikey in slot 2, type the following command:
 
-    ykpersonalize -2 -ochal-resp -ochal-hmac -ohmac-lt64 -oserial-api-visible
+    ykman otp chalresp -g 2
 
 This configures slot 2 for challenge-response, and leaves slot 1 alone.
 
-Next we need to create some challenge response files, move them to a system wide directory, and secure those files:
+Next we need to create a place to store your challenge response files, secure those files, and finally create the stored challenge files:
 
     sudo mkdir /var/yubico
-    sudo chown root.root /var/yubico
+    sudo chown root /var/yubico
     sudo chmod 700 /var/yubico
     ykpamcfg -2 -v
 
 You should receive a message similar to:
 
 `Stored initial challenge and expected response in '$HOME/.yubico/challenge-123456'.`
 
-You should receive a unique *challenge-123456* in your output.
+You should receive a unique *challenge-serial* in your output.
 
 Now, to finish up:
 
     sudo mv ~/.yubico/challenge-123456 /var/yubico/alice-123456
-    sudo chown root.root /var/yubico/alice-123456
-    sudo chmod 600 /var/yubico/alice-123456
+    sudo chown root.root /var/yubico/bob-123456
+    sudo chmod 600 /var/yubico/lisa-123456
```

Comment:
I'm confused by the usage of `alice` then `bob` then `lisa` in these examples. When I set my yubikey up, will I be changing the names in each command as I'm running them, like these examples show? If not, we should use only one name and keep it the same every time.

Also, further up, `123456` was changed to `serial` but it is still `123456` in these examples. Should that also be made more consistent?